packages/x-components/src/x-modules/url/store/types.ts

```diff
@@ -53,7 +72,14 @@ export interface UrlMutations {
  *
  * @public
  */
-export interface UrlActions {}
+export interface UrlActions {
+  /**
+   * It updates the URL with values from the store.
```

Comment:
Remove the `It`. Explain that it replaces the current URL without pushing a new entry to the browser history.